virt/virtual_machines/cloning_vms/virt-cloning-vm-disk-into-new-datavolume-block.adoc

```diff
@@ -16,6 +16,11 @@ Cloning operations between different volume modes are supported, such as cloning
 However, you can only clone between different volume modes if they are of the `contentType: kubevirt`.
 ====
 
+[TIP]
+====
+When you globally enable preallocation, the Containerized Data Importer preallocates disk space to improve write performance during cloning. See xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Using preallocation for data volumes] for more information.
```

Comment:
This reads a bit awkwardly to me.  "the Containerized Data Importer preallocates disk space to improve write performance during cloning" -- sounds like this is saying that write performance will be improved during the cloning process.  I think we want to phrase more like "When preallocation is enabled globally, CDI preallocates disk space during cloning.  For more information about preallocation including the benefit to write performance see \#link)

Reply:
Changed to: 

[TIP]
====
When preallocation is enabled globally, the containerized data importer (CDI) preallocates disk space during cloning. For more information, including benefits to write performance, see xref:../../../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[Using preallocation for data volumes].
====